Test/allocated1/dafny0/Twostate-Functions.dfy.expect

```diff
@@ -5,18 +5,18 @@ Twostate-Functions.dfy(70,17): Error: assertion might not hold
 Twostate-Functions.dfy(57,14): Related location
 Twostate-Functions.dfy(72,15): Error: assertion might not hold
 Twostate-Functions.dfy(57,14): Related location
-Twostate-Functions.dfy(96,24): Error: argument ('u') must be allocated in the two-state function's previous state
-Twostate-Functions.dfy(101,40): Error: argument 1 ('x') must be allocated in the two-state function's previous state
-Twostate-Functions.dfy(107,40): Error: argument 1 ('x') must be allocated in the two-state function's previous state
+Twostate-Functions.dfy(96,24): Error: argument ('u') may not be allocated in the two-state function's previous state
```

Comment:
"may not" implicitly means "does not have the authorization to", which is the opposite of the previous error message. Can you explain this or fix it? You might consider `might not`, which we use in other places.

Reply:
Oh, yeah, "might not" is much better. :)